tests/src/Kernel/fixtures/markup_rendering_patterns/pagination.yml

```diff
@@ -6,6 +6,7 @@ pagination:
       settings:
         size: 'sm'
         alignment: 'center'
+        list_attributes: 'class="list-class" id="list-id" data-list-attribute="list-attr-value"'
```

Comment:
As I said in previous reviews, do not change the default test. In here we should just test the default values.
Add a new one, with the extra attributes.

Reply:
did it in a separate test.